src/Aspire.Hosting/Extensions/ProjectResourceBuilderExtensions.cs

```diff
@@ -208,6 +222,17 @@ public static IResourceBuilder<ProjectResource> ExcludeLaunchProfile(this IResou
         throw new InvalidOperationException("This API is replaced by the AddProject overload that accepts a launchProfileName. Null means exclude launch profile. Method will be removed by GA.");
     }
 
+    /// <summary>
+    /// Configures the project to disable forwarded headers when being published.
+    /// </summary>
+    /// <param name="builder">The project resource builder.</param>
+    /// <returns>A reference to the <see cref="IResourceBuilder{T}"/>.</returns>
+    public static IResourceBuilder<ProjectResource> DisableForwadedHeadersOnPublish(this IResourceBuilder<ProjectResource> builder)
```

Comment:
Open to other naming suggestions, e.g.:
- `PublishWithForwardedHeaders(false)`
- `DisableForwardedHeaders()`
- `ExcludeForwardedHeaders()`
- `WithoutForwardedHeaders()`

Reply:
`DisableForwardedHeaders`